modules/fedimint-dummy/src/lib.rs

```diff
@@ -118,18 +117,16 @@ impl ModuleInit for DummyConfigGenerator {
         Ok(Ok(server.to_erased()))
     }
 
-    fn to_client_config(&self, config: ServerModuleConfig) -> anyhow::Result<ClientModuleConfig> {
-        Ok(config
-            .to_typed::<DummyConfig>()?
-            .consensus
-            .to_client_config())
-    }
-
-    fn to_client_config_from_consensus_value(
+    fn to_config_response(
         &self,
         config: serde_json::Value,
-    ) -> anyhow::Result<ClientModuleConfig> {
```

Comment:
It's not this PR specific, but we should require users to gtouch `serde_json::Value` in the implementations of these traits. We should have associated types for everything. Will create an issue.

Reply:
#1297

Generally when working with it I often get lost in how many types (split configs, aggregates configs) of the `serde_json::Value` we have, all named in rather similar way. Something to think about - maybe there's a way to improve it.